.github/workflows/backport.yml

```diff
@@ -0,0 +1,34 @@
+# Based on https://github.com/NixOS/nixpkgs/blob/master/.github/workflows/backport.yml
+
+name: Backport merged pull requests
+on:
+  pull_request_target:
+    types: [closed, labeled]
+
+permissions: {}
+
+jobs:
+  backport:
+    name: Backport pull request
+    if: github.repository_owner == 'rpm-software-management' && github.event.pull_request.merged == true && (github.event_name != 'labeled' || startsWith('backport', github.event.label.name))
```

Comment:
Doesn't <code>github.event_name != 'labeled' || startsWith('backport', github.event.label.name<code> disregard the label value when the pull request is closed? I.e. it executes korthout/backport-action@v3 for pull requests without a label.